source/messaging/channel-preferences.rst

```diff
@@ -18,53 +18,96 @@ Set Channel Preferences
 
-You can set channel preferences, such as notifications, as well as the channel's header, purpose, and name for each channel you belong to. To access these settings, select the channel name at the top of the page.
+For each channel you belong to, you can set notification preferences as well as the channel's header, purpose, and name. Select the channel name at the top of the screen to access channel-specific settings.
+
+.. image:: ../images/channel-preferences.png
+        :alt: Select the channel name at the top of the screen to set preferences for each channel you belong to.
 
 Channel notification preferences
 --------------------------------
 
-Each channel you belong to can have different notification preferences.
+Change the following channel notification settings by selecting **Notification Preferences**.
 
 Mute channel
 ~~~~~~~~~~~~~
 
-By default, channel muting is turned off for all channels. To mute or unmute a channel, select the channel name at the top of the page to access the channel menu, then choose **Mute Channel**.
+By default, all chanels are unmuted. To mute or unmute the current channel, enable the **Mute Channel** option, then select **Save**.
+
+.. tip::
+
+  Quickly mute or unmute any channel any time by selecting the channel name at the top of the screen and selecting **Mute Channel** or **Unmute Channel**.
 
 Ignore mentions for @channel, @here and @all
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-By default, you'll receive mention notifications for any uses of @channel, @all or @here in a channel. When enabled, the channel will ignore mention notifications for channel wide mentions. Any messages containing @channel, @all or @here will still mark the channel unread, unless channel mute is enabled.
+By default, you'll receive mention notifications every time someone on your team `mentions an entire channel <https://docs.mattermost.com/messaging/mentioning-teammates.html>`__ using ``@channel``, ``@all``, or ``@here``. 
+
+To stop receiving mention notifications, enable the **Ignore mentions for @channel, @here and @all** option, then select **Save**. When enabled, mention notifications for channel-wide mentions are ignored, but the channel is marked as unread unless the channel is muted. 
 
 Send desktop notifications
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-By default, the desktop notification preference assigned in **Settings** is used for all channels. To customize the desktop notification preference for each channel, select the channel name at the top of the page to access the channel menu, then select **Notification Preferences > Send desktop notifications**.
+By default, your `desktop notification preferences <https://docs.mattermost.com/messaging/manage-channels-settings.html#desktop-notifications>`__ set in **Settings** apply to all channels. Desktop notifications are available on Edge, Firefox, Safari, Chrome, and `Mattermost Desktop Apps <https://mattermost.com/download/#mattermostApps>`__.
```

Comment:
As a new use I might not know what "Settings" means, maybe link to related docs?